pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -239,8 +239,7 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo) (*apps.Dep
 			"--",
 			"--foreground",
 			"--id", osdID,
-			"--osd-uuid", osd.UUID,
-			"--conf", osd.Config,
```

Comment:
Is removing this `--conf` parameter the critical part of this change to fix the ceph overrides? If not, I'm not clear how this is fixing it.

Reply:
Right, actually I don't believe `--osd-uuid` is needed here that's why I removed it. It looks like a relic from the old-style OSD configuration.